docs/howto/openstack/nova/move-server-between-regions.md

```diff
@@ -0,0 +1,221 @@
+# Moving a server from one region to another
+
+This guide will show you how to move a server to a different region in {{brand}} Cloud.
+
+## Prerequisites
+
+- Installed [OpenStack CLI client](/howto/getting-started/enable-openstack-cli/)
+- Access to both regions RC-file
+- Enough space on the local machine that you download your server to
+- Target region already configured with Network, Security Groups etc
+
+## Finding a volume ID
+
+=== "{{gui}}"
+    To find your server's boot volume in the {{gui}}, open your web
+    browser, navigate to the [{{gui}}](https://{{gui_domain}}), and log into your {{brand}} account.
+
+    On the left side panel of the {{gui}} expand _Compute_, then click on _Servers_. 
+
+    ![left-side-panel](assets/move-server-between-regions/shot-01.png)
+
+    To get more details of the server you want to move, expand the view of the server by clicking it.
+
+    ![server](assets/move-server-between-regions/shot-02.png)
+
+    On the left bottom side of the expanded view, there is a section called _Attached volumes:_. Below this you will see _{- Boot Volume -} <num\>GB_, click on it and then click _Copy ID_.
+=== "Openstack CLI"
+    If you work with the OpenStack CLI, please do not forget to [source the RC file first](/howto/getting-started/enable-openstack-cli/).
+
+    Use the ID of the server instead of using the server name. This will make sure that you are using the correct server.
+
+    Find the IDs of your servers by matching the name, using the following command:
+
+    ```bash
+    openstack server list --name 'name'
+    ```
+
+    Printout reference:
+
+    ```plain
+    +-----------------+-----------------+---------+-----------------+--------------------+-------------+
+    | ID              | Name            | Status  | Networks        | Image              | Flavor      |
+    +-----------------+-----------------+---------+-----------------+--------------------+-------------+
+    | a51e3d79-....-. | Name2           | SHUTOFF | STO2_test_netwo |                    | b.1c1gb     |
+    | ...-....-...... |                 |         | rk=10.x.y.z     |                    |             |
+    | ......          |                 |         |                 |                    |             |
```

Comment:
Please don't replace UUIDs with dots. To mask real UUIDs with non-existent ones, just use `uuidgen`.